langchain/document_loaders/unstructured.py

```diff
@@ -165,30 +222,42 @@ class UnstructuredAPIFileIOLoader(UnstructuredFileIOLoader):
 
     def __init__(
         self,
-        file: IO,
+        file: Optional[IO] = None,
         mode: str = "single",
         url: str = "https://api.unstructured.io/general/v0/general",
         api_key: str = "",
+        files: Optional[List[IO]] = None,
+        file_filename: Optional[str] = "",
```

Comment:
could these be part of unstructured_kwargs?

Reply:
(as in file_filename and file_filenames)

Reply:
Yep those can just be `unstructured_kwargs`. I'll update that.